scripts/exglobal_atmos_tropcy_qc_reloc.sh

```diff
@@ -13,11 +13,11 @@
 # Make sure we are in the $DATA directory
 cd "${DATA}"
 
-tmhr=$(echo "${tmmark}"|cut -c3-4)
-cdate10=$( ${NDATE:?} -"${tmhr}" "${PDY}""${cyc}")
+tmhr=${tmmark:2:2}
+cdate10=$(date --utc +%Y%m%d%H -d "${PDY}${cyc} - ${tmhr} hours")
```

Comment:
```suggestion
cdate10=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${tmhr} hours")
```
Needs a space between YYYYmmdd and HH